otherlibs/sites_locations/src/plugins/dune

```diff
@@ -0,0 +1,9 @@
+(library
+ (name sites_locations_plugins)
```

Comment:
The name here looks quite long without telling the reader any additional information as far as I can tell. What about something simpler like `dune_site`?